src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -566,205 +874,240 @@ s32 func_80A122EC(EnGo* this) {
     return false;
 }
 
-void func_80A123A0(EnGo* this, PlayState* play) {
-    Vec3f sp2C;
+/**
+ * Collider Update for the Rolling Snowball form of the Gatekeeper
+ */
+void EnGo_UpdateCollider_Snowball(EnGo* this, PlayState* play) {
```

Comment:
```suggestion
void EnGo_Snowball_UpdateCollider(EnGo* this, PlayState* play) {
```
Looks like you used Snowball as the subsystem elsewhere, so this should match that style. Same for the other functions where you put UpdateCollider as the subsystem.

Reply:
I didn't quite mean to use Snowball as a subsystem. Snowball is an action, and I tried to structure the subsystems around the goron types, or the things that wouldn't likely be confused with the goron types (UpdateCollider, InitEffect, etc...)

I think I'd prefer changing the current "EnGo_Snowball_CanHurtPlayer" to "EnGo_CanSnowballHurtPlayer" to avoid that, and maybe change this function to "EnGo_UpdateSnowballCollider" (...UpdateRolledUpCollider, UpdateFrozenCollider, etc...), if that'd be acceptable.

Reply:
That would be fine if `snowball` and the others are not meant to be subsystems.